<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtCaptureDataView</class>
 <widget class="QWidget" name="QtCaptureDataView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1712</width>
    <height>940</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="configuration">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1501</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Configuration</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1481</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,200,0,200,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Evaluation Images: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QDoubleSpinBox" name="maxYawDoubleSpinBox">
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="evalImagesSpinBox">
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="maxRollLabel">
       <property name="text">
        <string>Max Roll: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="maxRollDoubleSpinBox">
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="trainImagesLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Train Images: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="bbAlgorithmLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Bounding Box Algorithm:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="trainImagesSpinBox">
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="maxPitchLabel">
       <property name="text">
        <string>Max Pitch: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Face Alignment Algorithm:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Name: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="bbAlgorithmComboBox"/>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="maxPitchDoubleSpinBox">
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="faAlgorithmComboBox"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="maxYawLabel">
       <property name="text">
        <string>Max Yaw: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>°</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>°</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>°</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="actions">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>20</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Tasks</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>160</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="initializeCameraPushButton">
       <property name="text">
        <string>Initialize Camera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startCapturePushButton">
       <property name="text">
        <string>Start Capture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="visualization">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>1691</width>
     <height>631</height>
    </rect>
   </property>
   <property name="title">
    <string>Visualization</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1671</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,1">
     <item row="0" column="0">
      <widget class="QCheckBox" name="visualizeCheckBox">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Visualize</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Camera RGB</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Face Alignment</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="imageColorLabel">
       <property name="text">
        <string>Image1</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="imageAlignmentLabel">
       <property name="text">
        <string>Image2</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="messages">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>800</y>
     <width>1691</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Status Messages</string>
   </property>
   <widget class="QTextBrowser" name="statusMessagesContainer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1671</width>
      <height>101</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>faAlgorithmComboBox</tabstop>
  <tabstop>trainImagesSpinBox</tabstop>
  <tabstop>maxPitchDoubleSpinBox</tabstop>
  <tabstop>evalImagesSpinBox</tabstop>
  <tabstop>maxRollDoubleSpinBox</tabstop>
  <tabstop>bbAlgorithmComboBox</tabstop>
  <tabstop>maxYawDoubleSpinBox</tabstop>
  <tabstop>initializeCameraPushButton</tabstop>
  <tabstop>startCapturePushButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
  <tabstop>visualizeCheckBox</tabstop>
  <tabstop>statusMessagesContainer</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>nameLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QtCaptureDataView</receiver>
   <slot>onConfigurationGroupChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trainImagesSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtCaptureDataView</receiver>
   <slot>onConfigurationGroupChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>evalImagesSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtCaptureDataView</receiver>
   <slot>onConfigurationGroupChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbAlgorithmComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>QtCaptureDataView</receiver>
   <slot>onConfigurationGroupChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>faAlgorithmComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>QtCaptureDataView</receiver>
   <slot>onConfigurationGroupChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1485</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxPitchDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QtCaptureDataView</receiver>
   <slot>onConfigurationGroupChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1487</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxRollDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QtCaptureDataView</receiver>
   <slot>onConfigurationGroupChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1461</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxYawDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QtCaptureDataView</receiver>
   <slot>onConfigurationGroupChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1487</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>initializeCameraPushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCaptureDataView</receiver>
   <slot>onInitializeCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1531</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>856</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startCapturePushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCaptureDataView</receiver>
   <slot>onStartCapture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1531</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>952</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCaptureDataView</receiver>
   <slot>onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1531</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>954</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>visualizeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QtCaptureDataView</receiver>
   <slot>onVisualizationGroupChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConfigurationGroupChanged()</slot>
  <slot>onInitializeCamera()</slot>
  <slot>onStartCapture()</slot>
  <slot>onCancel()</slot>
  <slot>onVisualizationGroupChanged()</slot>
 </slots>
</ui>
